src/vtcomposite.cpp

```diff
@@ -1035,33 +1035,25 @@ Napi::Value localize(Napi::CallbackInfo const& info)
         Napi::Value language_val = params.Get(Napi::String::New(info.Env(), "languages"));
         if (language_val.IsArray())
         {
+            return_localized_tile = true;
+
             Napi::Array language_array = language_val.As<Napi::Array>();
             std::uint32_t num_languages = language_array.Length();
 
-            if (num_languages > 0)
+            for (std::uint32_t lg = 0; lg < num_languages; ++lg)
             {
-                languages.reserve(num_languages);
```

Comment:
why we don't need languages.reserve anymore?

Reply:
Restore in [fecd436](https://github.com/mapbox/vtcomposite/pull/131/commits/fecd436a4f90222b756b4f9ad0c156ed3c118e47).